Build fails on Windows because of 2 failing TestOriginLinkFormatter test cases

# Description
When executing `mvn install` on Windows the build fails because of 2 failing TestOriginLinkFormatter test cases.

## Steps to Reproduce
1. Open a CMD or Git Bash in the directory where OFT was cloned to
2. Execute `mvn install`

## Expected behavior
Build finishes successfully

# Environment
- OFT: 2.0.1
- OS: Windows 7 x64
- Java Version: 1.8.0_181
- Maven Version (in case of build problems only): 3.5.4

## Error messages
```
[ERROR] testFormatRegularAbsoluteUnixPath(org.itsallcode.openfasttrace.report.view.html.TestOriginLinkFormatter)  Time elapsed: 0.001 s  <<< FAILURE!
java.lang.AssertionError:

Expected: "<span class=\"origin\"><a href=\"file:///foo/bar/baz\">/foo/bar/baz</a>:1111</span>"
     but: was "<span class=\"origin\">/foo/bar/baz:1111</span>"
        at org.itsallcode.openfasttrace.report.view.html.TestOriginLinkFormatter.assertPathAndLineRenderedToSpan(TestOriginLinkFormatter.java:57)
        at org.itsallcode.openfasttrace.report.view.html.TestOriginLinkFormatter.testFormatRegularAbsoluteUnixPath(TestOriginLinkFormatter.java:85)
```

```
[ERROR] testFormatAbsoluteUnixPathWithSpecialCharacters(org.itsallcode.openfasttrace.report.view.html.TestOriginLinkFormatter)  Time elapsed: 0 s  <<< FAILURE!
java.lang.AssertionError:

Expected: "<span class=\"origin\"><a href=\"file:///fo%20o/b%C3%A4r/baz\">/fo o/bär/baz</a>:12345678</span>"
     but: was "<span class=\"origin\">/fo o/bär/baz:12345678</span>"
        at org.itsallcode.openfasttrace.report.view.html.TestOriginLinkFormatter.assertPathAndLineRenderedToSpan(TestOriginLinkFormatter.java:57)
        at org.itsallcode.openfasttrace.report.view.html.TestOriginLinkFormatter.testFormatAbsoluteUnixPathWithSpecialCharacters(TestOriginLinkFormatter.java:92)
```

--- a/src/main/java/org/itsallcode/openfasttrace/report/view/html/OriginLinkFormatter.java
+++ b/src/main/java/org/itsallcode/openfasttrace/report/view/html/OriginLinkFormatter.java
@@ -138,6 +138,6 @@ public final class OriginLinkFormatter
 
     private static String unWindowsify(final String path)
     {
-        return path.replaceAll("([A-Z]:)", "/$1").replaceAll("\\\\", "/");
+        return path.replaceAll("\\\\", "/");
     }
 }

--- a/src/test/java/org/itsallcode/openfasttrace/report/view/html/TestOriginLinkFormatter.java
+++ b/src/test/java/org/itsallcode/openfasttrace/report/view/html/TestOriginLinkFormatter.java
@@ -28,6 +28,7 @@ import static org.itsallcode.openfasttrace.report.view.html.OriginLinkFormatter.
 import static org.junit.Assert.assertThat;
 
 import org.itsallcode.openfasttrace.core.Location;
+import org.itsallcode.openfasttrace.testutil.OsDetector;
 import org.junit.Test;
 
 public class TestOriginLinkFormatter
@@ -82,15 +83,33 @@ public class TestOriginLinkFormatter
     @Test
     public void testFormatRegularAbsoluteUnixPath()
     {
+        OsDetector.assumeRunningOnUnix();
         assertPathAndLineRenderedToSpan("/foo/bar/baz", 1111,
                 "<a href=\"file:///foo/bar/baz\">/foo/bar/baz</a>:1111");
     }
 
     @Test
+    public void testFormatRegularAbsoluteWindowsPath()
+    {
+        OsDetector.assumeRunningOnWindows();
+        assertPathAndLineRenderedToSpan("C:\\foo\\bar\\baz", 1111,
+                "<a href=\"file:///C:/foo/bar/baz\">C:\\foo\\bar\\baz</a>:1111");
+    }
+
+    @Test
     public void testFormatAbsoluteUnixPathWithSpecialCharacters()
     {
+        OsDetector.assumeRunningOnUnix();
         assertPathAndLineRenderedToSpan("/fo o/bär/baz", 12345678,
                 "<a href=\"file:///fo%20o/b%C3%A4r/baz\">/fo o/bär/baz</a>:12345678");
+    }
+
+    @Test
+    public void testFormatAbsoluteWindowsPathWithSpecialCharacters()
+    {
+        OsDetector.assumeRunningOnWindows();
+        assertPathAndLineRenderedToSpan("C:\\fo o\\bär\\baz", 12345678,
+                "<a href=\"file:///C:/fo%20o/bär/baz\">C:\\fo o\\bär\\baz</a>:12345678");
     }
 
     @Test

